src/helpers/responseHandler.js

```diff
@@ -0,0 +1,38 @@
+/**
+* class for responses
+ */
+class response {
+  /**
+   * getting response
```

Comment:
### Nitpick

- Not descriptive enough

Reply:
I resolved it by adding more descriptive